src/cron/utils/read-markdown-messages.js

```diff
@@ -0,0 +1,116 @@
+const { scheduleMessages } = require('../schedule-messages');
+const fs = require('fs');
+const path = require('path');
+const { SCHEDULE_MESSAGES } = require('../../config');
+
+/**
+ * Function to search for all Markdown files in a directory.
+ * @param {string} directory - Path to the directory to scan.
+ * @returns {string[]} - List of Markdown file paths.
+ */
+const getMarkdownFiles = (directory) => {
+  const markdownFiles = [];
+
+  try {
+    const files = fs.readdirSync(directory);
+
+    files.forEach((file) => {
+      const filePath = path.join(directory, file);
+      const stats = fs.statSync(filePath);
+
+      if (stats.isFile() && file.endsWith('.md')) {
+        markdownFiles.push(filePath);
+      }
+    });
+  } catch (error) {
+    console.error(`Error reading the directory "${directory}":`, error.message);
+  }
+
+  return markdownFiles;
+};
+
+/**
+ * Function to read the content of a Markdown file.
+ * @param {string} filePath - Path to the file to read.
+ * @returns {string|null} - File content or null if an error occurs.
+ */
+const readMarkdownFile = (filePath) => {
+  try {
+    return fs.readFileSync(filePath, 'utf8');
+  } catch (error) {
+    console.error(`Error reading the file "${filePath}":`, error.message);
+    return null;
+  }
+};
+
+/**
+ * Function to parse a Markdown file with a specific format.
+ * @param {string} markdown - Content of the Markdown file.
+ * @returns {{variables: object, message: string}|null} - Variables and message, or null if errors occur.
+ */
+const parseMarkdownSchedule = (markdown) => {
+  const lines = markdown.split('\n');
+  const variables = {};
+  let message = '';
+  let isVariableDefinition = false;
+  let errorOccurred = false;
+
+  for (const line of lines) {
+    const trimmedLine = line.trim();
+
+    if (trimmedLine === '---') {
+      isVariableDefinition = !isVariableDefinition;
+      continue;
+    }
+
+    if (isVariableDefinition) {
+      const [key, value] = trimmedLine.split(':').map((str) => str.trim());
+      if (!key || value === undefined) {
+        console.error(`Syntax error on line: "${line}"`);
+        errorOccurred = true;
+        continue;
+      }
+      variables[key] = value;
+    } else {
+      message += `${line}\n`;
+    }
+  }
+
+  if (errorOccurred) {
+    console.error(
+      'Errors found in variable definitions. Please fix them before continuing.'
+    );
+    return null;
+  }
+
+  return { variables, message: message.trim() };
+};
+
+/**
+ * Main function to process all Markdown files in a directory.
+ * @param {string} directory - Path to the directory containing the Markdown files.
+ */
+const processMarkdownFiles = (client) => {
+  const absoluteDirectory = path.resolve(
+    __dirname,
```

Comment:
instead of __dirname you could use process.cwd() https://www.geeksforgeeks.org/node-js-process-cwd-method/

so you will have the path to the root of the project

here is an example: https://github.com/heliomar-pena/i18n-populator/blob/master/utils/getConfigPath.js
![image](https://github.com/user-attachments/assets/84018860-cd53-4eb7-8da3-7fb0038adbb5)

Reply:
doing it this way you can avoid the relative path `../../../` that you're doing here: https://github.com/TeamNovaSoft/discord-bot/pull/83/files#diff-23821ae067a9c7dfb6227774aefb39fa3951a5874ea953b9401dfc80175c026cR47